source/painters/systemrenderer.cpp

```diff
@@ -1653,19 +1669,24 @@ void SystemRenderer::drawIrregularGroups(const Voice &voice,
         // vertical lines on either end.
         QGraphicsLineItem *horizLine1 = new QGraphicsLineItem();
         horizLine1->setLine(leftX, y2, leftX + lineWidth, y2);
+        horizLine1->setPen(QPen(myPalette.light().color()));
```

Comment:
Should this be `text()` instead of `light()` to match the other lines here?

Reply:
I agree, I will fix this